include/swift/SIL/TypeSubstCloner.h

```diff
@@ -109,7 +109,11 @@ class TypeSubstCloner : public SILClonerWithScopes<ImplClass> {
 
       assert(Subs.empty() ||
              SubstCalleeSILType ==
-                 Callee->getType().substGenericArgs(AI.getModule(), Subs));
```

Comment:
This assertion is a bit silly now, how about just removing it instead of teaching TypeSubstCloner about opaque types